service/s3/endpoint_test.go

```diff
@@ -112,18 +112,46 @@ func TestEndpoint(t *testing.T) {
 			bucket: "arn:aws-us-gov:s3:us-gov-west-1:123456789012:accesspoint:myendpoint",
 			config: &aws.Config{
 				Region: aws.String("fips-us-gov-west-1"),
+				EndpointResolver: endpoints.ResolverFunc(
+					func(service, region string, opts ...func(*endpoints.Options)) (endpoints.ResolvedEndpoint, error) {
+						switch region {
+						case "fips-us-gov-west-1":
+							return endpoints.ResolvedEndpoint{
+								URL:           "s3-fips.us-gov-west-1.amazonaws.com",
+								PartitionID:   endpoints.AwsUsGovPartition().ID(),
```

Comment:
This might fail to compile if the SDK generated uses a custom endpoint that does not contain the UsGovPartition Endpoint group.

Reply:
updated